packages/engine/src/behave-graph/nodes/Profiles/Engine/Values/CustomNodes.ts

```diff
@@ -212,7 +214,7 @@ export const getAvatarAnimations = makeFunctionNodeDefinition({
 })
 
 export const playGltfAnimation = makeFlowNodeDefinition({
-  typeName: 'engine/playGltfAnimation',
+  typeName: 'engine/media/playGltfAnimation',
```

Comment:
we can rename playGltfAnimation to just playAnimation

Reply:
done